src/v2/Apps/ArtistSeries/Components/ArtistSeriesHeader.tsx

```diff
@@ -1,14 +1,74 @@
 import React from "react"
-import { Box, Col, Flex, Grid, Row, Sans } from "@artsy/palette"
+import {
+  Box,
+  Col,
+  EntityHeader,
+  Flex,
+  Grid,
+  Row,
+  Sans,
+  Separator,
+} from "@artsy/palette"
 import { Media } from "v2/Utils/Responsive"
 import { createFragmentContainer, graphql } from "react-relay"
-
+import { FollowArtistButtonFragmentContainer as FollowArtistButton } from "v2/Components/FollowButton/FollowArtistButton"
+import { openAuthToFollowSave } from "v2/Utils/openAuthModal"
 import { ArtistSeriesHeader_artistSeries } from "v2/__generated__/ArtistSeriesHeader_artistSeries.graphql"
+import { useSystemContext } from "v2/Artsy"
+import { Intent } from "@artsy/cohesion"
 
 interface ArtistSeriesHeaderProps {
   artistSeries: ArtistSeriesHeader_artistSeries
 }
 
+interface ArtistsInfoProps {
+  artist: ArtistSeriesHeader_artistSeries["artists"][0]
+}
+
+const ArtistInfo: React.FC<ArtistsInfoProps> = props => {
+  /* Displays artist name, avatar and follow button. We currently assume
+     that an artist series will have one artist. */
+  const { user, mediator } = useSystemContext()
+  const { artist } = props
+  return (
+    <EntityHeader
+      smallVariant
+      name={artist.name}
+      imageUrl={artist.image?.url}
+      href={artist.href}
+      FollowButton={
+        <FollowArtistButton
+          artist={artist}
+          user={user}
+          onOpenAuthModal={() =>
+            openAuthToFollowSave(mediator, {
+              entity: artist,
+              // FIXME: Add artist series to Cohesion
+              contextModule: null,
+              intent: Intent.followArtist,
+            })
+          }
+          render={({ is_followed }) => {
+            return (
+              <Sans
+                size="3"
+                color="black"
+                data-test="followArtistButton"
+                style={{
+                  cursor: "pointer",
+                  textDecoration: "underline",
+                }}
+              >
+                {is_followed ? "Following" : "Follow"}
+              </Sans>
```

Comment:
Is this the idiomatic way we render these follow links?

Occurs to me that this is neither an `<a>` nor a `<button>`, which may have accessibility implications (cc @dzucconi) such as tab order etc.

Reply:
That's a good point! I found this pattern elsewhere in the codebase so I used that. We should modify it though if it's not accessible.

Reply:
It looks like you'd have to alter https://github.com/artsy/force/blob/784eeca562f560dd4e37d952fc28c2800a4b8221/src/v2/Components/FollowButton/FollowArtistButton.tsx#L166 that span should be a `Clickable`.

Reply:
Cool -- since that would modify a component used in many other places, I'd rather take that as a follow up item than do it in this PR. I'll make a ticket.